### PR TITLE
LWIMIQ Survey Says 2017-10-23

### PR DESCRIPTION
**Pull request by JPrause**

### CODE REVIEW

**Comment by JPrause** [comment]
Since we had an opening this week, I took the opportunity to provide our community users with a summary of the survey results. Got this in early since it's pretty lengthy.

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@syncrou @imtayadeway @jntullo can you review as time allows for this week's blog post.

File: site/_posts/2017-10-23-survey-says.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - Visual (spatial):You prefer using pictures, images, and spatial understanding.
+    - Aural (auditory-musical): You prefer using sound and music.
+    - Verbal (linguistic): You prefer using words, both in speech and writing.
+    - Physical (kinesthetic): You prefer using your body, hands and sense of touch.
+    - Logical (mathematical): You prefer using logic, reasoning and systems.
+    - Social (interpersonal): You prefer to learn in groups or with other people.
+    - Solitary (intrapersonal): You prefer to work alone and use self-study.
+
+Understanding your [learning style][learn] can increase how much information you process and are able to retain. As an example, I will never understand how some people can read a book while music is blaring in the background, but I digress. We as a team are constantly trying to improve our methods of communicating information about ManageIQ. Whether by direct instruction, [videos][vids], or [documentation][doc]. As stated before, this is an on-going journey and we are aware that we need to use or enhance our methods of communication.
```
Comment (imtayadeway) [inline site/_posts/2017-10-23-survey-says.md]: Citation needed :trollface:

Comment (JPrause) [reply site/_posts/2017-10-23-survey-says.md]: Good point! Coming up sir!

**Comment by JPrause** [comment]
@cybette can you have a peek and merge if all looks good.

**Comment by JPrause** [comment]
@chessbyte and @Fryguy here's the blog post for this week's LWIMIQ that covers the recent community survey results. Just a FYI or if you'd also like to review.

File: site/_posts/2017-10-23-survey-says.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  - Do you know how to contribute to the ManageIQ open source project?
+  - Is there anything else you’d like to share about ManageIQ?
+
+We had 38 people respond, and the comments and suggestions were very detailed and extremely helpful. We condensed the responses down to the Top 4 issues and our response:
+  - **Performance** - This is something our team is constantly working on, as all software developers should be. We see this as a journey that should never end, should be constantly be tested, and should be constantly reviewed. As such, Jason Frey (aka [Fryguy][fry]) recently posted a blog article regarding the [H-Release Rearchitecture][hrelease]. In this article you'll find that (as he says); _**The primary reason**_ for the rearchitecture _**was due to an increasing number of customer escalations**_ and _**problems of scalability and performance**_. I don't do his blog article any justice by showing snippets. Go read the article in its entirety and you'll see the commitment we have for you, our community users. Jason will also _be following up with some more blog posts detailing some of the many other rearchitecture decisions_.
```
Comment (cybette) [inline site/_posts/2017-10-23-survey-says.md]: minor: Should be constantly ~~be~~ tested

It's a great post! I feel bad for nitpicking, but I'll merge right after this little little fix. Thanks @JPrause !

**Pull request merged**